transport-native-kqueue/src/main/c/netty_kqueue_native.c

```diff
@@ -381,69 +379,26 @@ static void netty_kqueue_native_JNI_OnUnLoad(JNIEnv* env, const char* staticPack
     netty_kqueue_eventarray_JNI_OnUnLoad(env, staticPackagePrefix);
 }
 
-static jint JNI_OnLoad_netty_transport_native_kqueue0(JavaVM* vm, void* reserved) {
-    JNIEnv* env;
-    if ((*vm)->GetEnv(vm, (void**) &env, NETTY_JNI_VERSION) != JNI_OK) {
-        return JNI_ERR;
-    }
-
-    char* packagePrefix = NULL;
-#ifndef NETTY_BUILD_STATIC
```

Comment:
looks like this changed to [NETTY_JNI_UTIL_BUILD_STATIC](https://github.com/netty/netty-jni-util/blob/main/src/c/netty_jni_util.c#L418)? be sure to call this out in release notes.

Reply:
I did ensure to define  NETTY_JNI_UTIL_BUILD_STATIC if NETTY_BUILD_STATIC is defined . so it will just work as before